shared/run_diff.go

```diff
@@ -42,10 +42,13 @@ func NewDiffAPI(ctx context.Context) DiffAPI {
 
 func (d diffAPIImpl) GetDiffURL(before, after TestRun, diffFilter *DiffFilterParam) *url.URL {
 	filter := TestRunFilter{}
-	filter.Products = ProductSpecs{
-		ProductSpec{ProductAtRevision: before.ProductAtRevision},
-		ProductSpec{ProductAtRevision: after.ProductAtRevision},
-	}
+	filter.Products = make(ProductSpecs, 2)
+	filter.Products[0].BrowserName = before.BrowserName
+	filter.Products[0].Revision = before.Revision
+	filter.Products[0].Labels = mapset.NewSet(before.Channel())
```

Comment:
What about including all the labels?

Reply:
Sigh. Let's land https://github.com/web-platform-tests/wpt.fyi/pull/906 and then it doesn't matter at all.